percona-server-mongodb-6.0/Dockerfile.k8s

```diff
@@ -68,8 +68,10 @@ RUN set -ex; \
     chown -R 1001:0 /data/db
 
 # the numeric UID is needed for OpenShift
-RUN useradd -u 1001 -r -g 0 -s /sbin/nologin \
+RUN useradd -u 1001 -r -g mongod -m -s /sbin/nologin \
             -c "Default Application User" mongodb
+# allow access to /var/log/mongo for mongodb user
+RUN chmod g+rwx /var/log/mongo
```

Comment:
maybe it is better to put this into one layer.
Decreasing amount of layers can decrease image size

Reply:
Done